portal/templates/index.njk

```diff
@@ -33,6 +33,17 @@
 
     <div class="govuk-width-container">
 
+      {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+      {% if backLink and backLink.href %}
+        {{ govukBackLink({
```

Comment:
Are you able to add a screenshot of how this looks now? Always useful to include a screenshot for visual changes so people can more easily comment on the design as well as the technical implementation.